examples/src/main/java/org/bitcoinj/examples/ForwardingService.java

```diff
@@ -171,4 +163,29 @@ private void forwardCoins() {
             throw new RuntimeException(e);
         }
     }
+
+    static CliArguments parseArguments(String[] args) {
```

Comment:
I was already wondering if we should introduce picocli for the example(s)?

Reply:
I would recommend against it for the following reasons:

* Most examples only have one or two parameters (I think some have zero)
* **picocli** adds some cognitive load to people who aren't familiar with it and distracts from the example
* **picocli** will tempt us to try to get too fancy when the point of the examples is to demonstrate how to use **bitcoinj**

Note that I was experimenting with a Kotlin version of this app last night and in Kotlin we are able to use the `Pair` type and get:
```
val (network, forwardingAddress ) = parseArguments(args)
```

Reply:
…however I feel the new `CliArguments` class distracts too. Are you sure this commit is an enhancement?

Reply:
Well, it shrinks `main()` by about 10 lines. Maybe the better solution is what I refer to here:

> a future PR that leaves the parameter parsing in main() and moves the actual async processing to another method

Reply:
I think this one does the job: https://github.com/bitcoinj/bitcoinj/pull/2535